src/node/coinstats.h

```diff
@@ -29,6 +30,6 @@ struct CCoinsStats
 };
 
 //! Calculate statistics about the unspent transaction output set
-bool GetUTXOStats(CCoinsView* view, CCoinsStats& stats);
+bool GetUTXOStats(CCoinsView* view, CCoinsStats& stats, const std::function<void()>& interruption_point = {});
```

Comment:
Is default ` = {}` really needed?

Reply:
Yes, otherwise all callers are forced to specify an interruption op

Reply:
See for example:

```
$ git grep --extended-regexp  'GetUTXOStats\([^,]+,[^,]+\)'
src/test/fuzz/coins_view.cpp:                (void)GetUTXOStats(&coins_view_cache, stats);